web/ui/react-app/.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "extends": [
+      "react-app",
+      "plugin:@typescript-eslint/recommended",
+      "plugin:prettier/recommended"
+    ],
+    "rules": {
+      "@typescript-eslint/camelcase": "warn",
+      "eol-last": ["error", "always"],
+      "object-curly-spacing": ["error", "always"],
+      "prefer-const": "warn",
+      "comma-dangle": ["error", {
+        "arrays": "always-multiline",
+        "objects": "always-multiline",
+        "imports": "always-multiline"
+      }]
+    },
+    "plugins": [
+      "prettier"
+    ]
+  }
+  
```

Comment:
Now the file ends with a space, not a newline :P

Reply:
Whoops, have removed it, I tried to get the linter to check it but seems to be a bit of a PITA to get it to do its own config